packages/keycloak-user-management/src/components/UserList/index.tsx

```diff
@@ -82,10 +82,8 @@ const UserList = (props: Props): JSX.Element => {
       serve
         .list()
         .then((res: KeycloakUser[]) => {
-          if (isLoading) {
-            setIsLoading(false);
-            fetchKeycloakUsersCreator(res);
-          }
+          setIsLoading(false);
```

Comment:
Best for this would be to add it in the `finally` block?

Reply:
@kelvin-muchiri  I would think so